forge/postoffice/index.js

```diff
@@ -11,6 +11,11 @@ module.exports = fp(async function (app, _opts, next) {
     let poStartupCheck
     const isConfigured = EMAIL_ENABLED && (app.config.email.smtp || app.config.email.transport || app.config.email.ses)
     const mailDefaults = { from: app.config.email.from ? app.config.email.from : '"FlowForge Platform" <donotreply@flowforge.com>' }
+    app.addHook('onClose', async (_) => {
+        if (poStartupCheck) {
+            clearInterval(poStartupCheck)
+        }
```

Comment:
```suggestion
            clearInterval(poStartupCheck)
```

Only a suggestion (dont need to test `poStartupCheck` is "something" as `clearInterval` will not throw.

 
![image](https://user-images.githubusercontent.com/44235289/232559904-bb74fa8a-5f1b-475a-8db2-df66f985d4d2.png)